pkg/image/admission/imagepolicy/imagepolicy_test.go

```diff
@@ -406,6 +454,9 @@ func TestAdmissionResolution(t *testing.T) {
 	if err := p.Admit(failingAttrs); err == nil {
 		t.Fatal(err)
 	}
+	if err := p.Validate(failingAttrs); err == nil {
+		t.Fatal(err)
```

Comment:
needs a better log statement, this is going to fail the test with err reason "nil"

Reply:
(copied from the one above which should also be fixed...)

Reply:
it'll include the line number, which gets you to the failed case... I'd rather keep the changes obviously additive than start changing the existing tests